deploy/apecloud-mysql-cluster/values.yaml

```diff
@@ -2,66 +2,54 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 3
+name: mycluster
 
-terminationPolicy: Delete
+## @param version MySQL cluster version
+##
+version: ac-mysql-8.0.30
 
-clusterVersionOverride: ""
-nameOverride: ""
-fullnameOverride: ""
+## @param mode MySQL cluster topology mode, standalone, replication or raftGroup
+##
+mode: standalone
 
-monitor:
-  enabled: false
+## @param replicas MySQL cluster replicas, for standalone mode, replicas must be 1
+replicas: 1
```

Comment:
`mode` and `replicas` seems clashed, do document override behavior?

Reply:
I will add corresponding documents when `kbcli` use this chart to create cluster. For this chart, I added some comments for this field.